energy-integration/fees-collector/tests/fees_collector_rust_test.rs

```diff
@@ -182,6 +182,113 @@ fn claim_first_week_test() {
         .assert_ok();
 }
 
+#[test]
+fn claim_after_dex_inactive_test() {
```

Comment:
Some of the comments are misleading, especially where week numbers are mentioned.